WooCommerce/Classes/ViewRelated/Blaze/BlazeEditAd/BlazeEditAdView.swift

```diff
@@ -252,6 +255,16 @@ private extension BlazeEditAdView {
                 value: "Change image",
                 comment: "Change image button title in the Blaze Edit Ad screen."
             )
+            static let imageSizeError = NSLocalizedString(
+                "blazeEditAdView.image.imageSizeError",
+                value: "Please select an image with minimum dimensions of 600 * 600 px.",
```

Comment:
We should use "× " instead of "*" for user-facing copy (the same symbol is used on mac).

```suggestion
                value: "Please select an image with minimum dimensions of 600 × 600 px.",
```

Reply:
I'll merge the change above and enable automerge for the code freeze.

Reply:
Thank you! ❤️